src/App.jsx

```diff
@@ -8,18 +8,24 @@ import CreateSubscription from "./components/CreateSubscription";
 import LoginForm from "./components/LoginForm";
 import SignUpForm from "./components/SignUpForm";
 import { connect } from "react-redux";
+import { getPlace } from "./modules/location";
 
-const App = () => {
+const App = (props) => {
   const [uid, setUid] = useState("");
   const [authenticated, setAuthenticated] = useState(false);
 
+  useEffect(() => {
+    getPlace(props.dispatch);
```

Comment:
I would say that we stick to hooks and use useDispatch instead here, since we use useSelector in ArticleList. Then you don't have to connect App either, I think.

Reply:
done